lib/inspec/waiver_file_reader.rb

```diff
@@ -20,9 +20,12 @@ def self.read_waivers_from_file(profile_id, files)
 
         if data.nil?
           raise Inspec::Exceptions::WaiversFileNotReadable,
-              "Cannot find parser for waivers file '#{file_path}'. " \
+              "Cannot find parser for waivers file." \
```

Comment:
Why are we removing file_path from this message? Any reason?

Reply:
That will be taken care here now https://github.com/inspec/inspec/pull/6644/files#diff-840ef8435a8ecac3b53d637518402eec237eaf326ca7e89c80f1c596b7d24148R27

Reply:
Yeah I see now, you did that to handle both `Inspec::Exceptions::WaiversFileNotReadable` and `Inspec::Exceptions::WaiversFileInvalidFormatting` errors in one place.